whalesong/driver_firefox.py

```diff
@@ -55,6 +58,17 @@ def __init__(self, profile: str = None, *,
 
         self.interval = interval
 
+    def free_port(self):
```

Comment:
It should be a async method, maybe it should use https://docs.python.org/3/library/asyncio-stream.html#asyncio.start_server